docs/src/Touchable.doc.js

```diff
@@ -94,9 +103,30 @@ card(
         name: 'onTouch',
         type:
           '({ event: SyntheticMouseEvent<HTMLDivElement> | SyntheticKeyboardEvent<HTMLDivElement> }) => void',
-        required: true,
         href: 'basicExample',
       },
+      {
+        name: 'pressBackgroundEnabled',
```

Comment:
@jackhsu978 we are trying to avoid boolean props as much as possible. Should we use `PressState` similar to what we use in Pinterest's codebase?

```
type PressState = 'border' | 'background' | 'compress' | 'none';
```

Reply:
+1 on this

Reply:
The reason I went with independent boolean props rather than just one value is that there are 31 places in Pinboard where we use both "background" and "compress":
```
pressState={['compress', 'background']}
```
there are also places where we only use "compress" and only use "background".

Alternatively, I could also do this instead:
```
type PressState = 'border' | 'background' | 'compress' | 'compressAndBackground'

pressState?: PressState,
```
Would that be better?

Reply:
what about: `pressState: PressState | Array<PressState>,`?

Reply:
The (very minor) issues with passing an array are:

- we might be inviting more unnecessary re-render of Touchable because the array reference is new each time parent renders
- an array can have duplicated elements

Thoughts?

Reply:
> we might be inviting more unnecessary re-render of Touchable because the array reference is new each time parent renders

As long as we don't have side-effects based on these re-renders, I highly doubt this will be a problem. It's technically okay to set a contract with the parent component to maintain memoization of params if we really want stability. I'm not saying we should or shouldn't go with an array, but just adding my 2 cents that re-rendering probably shouldn't be a big concern.

Reply:
thanks for the suggestions! updated the PR!